ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -171,7 +171,11 @@ SVFrequecyHeaderFilter.propTypes = {
 const QS_FILTER_FIELD = {
   name: 'min_qs',
   label: 'SV Quality Score',
-  labelHelp: 'The quality score (QS) represents the quality of a Structural Variant call. Recommended SV-QS cutoffs for filtering: duplications > 50; deletions > 100; homdel (CN=0) > 400',
+  labelHelp: (
+    <span>The quality score (QS) represents the quality of a Structural Variant call. Recommended SV-QS cutoffs for filtering:<br />
+      {'1) on Exome data: duplication >=50, deletion>=100 and homozygous deletion>=400;'}<br />
```

Comment:
numbering this will look weird, and text should just literal text not strings in brackets. Just do
```
WGS:  &gt; 10 <br />
WES: duplication &gt 50, deletion &gt 100, homozygous deletion &gt 400
```